mmf/models/base_model.py

```diff
@@ -170,7 +170,7 @@ def __call__(self, sample_list, *args, **kwargs):
             assert isinstance(
                 model_output["losses"], collections.abc.Mapping
             ), "'losses' must be a dict."
-        else:
+        elif hasattr(self, "losses"):
```

Comment:
Does this cause issues? This might cause issues down the line in the trainer where it won't find the losses key to sum upon. One option is to add a final case where you add empty dict for value key "losses" here?